.env.sample

```diff
@@ -15,8 +15,6 @@ INVOKE_LOG_STDOUT=true
 
 DJANGO_SETTINGS_MODULE={{project_name}}.settings
 GEONODE_INSTANCE_NAME=geonode
-GEONODE_LB_HOST_IP=
-GEONODE_LB_PORT=
```

Comment:
why did you remove these @marthamareal?

Reply:
Because they are duplicates. [here](https://github.com/GeoNode/geonode-project/blob/master/.env.sample#L67)

Reply:
Because they are duplicates. [here](https://github.com/GeoNode/geonode-project/blob/master/.env.sample#L67)